config/templates/atc.template.yml

```diff
@@ -1,4 +1,20 @@
 development:
   run_queued_jobs_inline: true
+  source_paths_to_storage_providers:
+    '/digital/preservation/':
+      path_mapping: '' # replace original path prefix with this value when transferring
+      storage_providers:
```

Comment:
Not required now, but I'm curious: what would  be the 'container_name' for a storage provider of type 'cul'?

Reply:
Good question.  Possibly something like "cul1" or "cul2", but the "cul"-bound ATC transfer use cases aren't too clear right now, and it would be good to revisit once we have more solid use cases for cul targets.